.rubocop.yml

```diff
@@ -103,7 +103,10 @@ Naming/HeredocDelimiterNaming:
   Enabled: false
 
 Naming/UncommunicativeMethodParamName:
-  Enabled: false
+  Enabled: true
+  AllowedNames:
+    - as
```

Comment:
I guess you can add some explanation about each entry.

Reply:
Sure thing ✅